diff --git a/node_modules/dsteem/lib/client.js b/node_modules/dsteem/lib/client.js
index c75c15c..7134543 100644
--- a/node_modules/dsteem/lib/client.js
+++ b/node_modules/dsteem/lib/client.js
@@ -124,12 +124,11 @@ class Client {
             const opts = {
                 body,
                 cache: 'no-cache',
-                headers: { 'User-Agent': `dsteem/${version_1.default}` },
                 method: 'POST',
                 mode: 'cors',
             };
             if (this.options.agent) {
-                opts.agent = this.options.agent;
+                opts.agent = this.options.agent || "dsteem/" + version_1["default"];
             }
             let fetchTimeout;
             if (api !== 'network_broadcast_api' && method.substring(0, 21) !== 'broadcast_transaction') {
